### PR TITLE
Add/fallback page

### LINKED ISSUE #3917: Admin: No JS view is broken
I turned off JS in my browser (Chrome), and don't get anything displaying at all. I think something went wrong in a recent merge or something.

### PR DESCRIPTION
**Pull request by zinigor**
Fixes #3917 and replaces #3806.

#### Changes proposed in this Pull Request:
- This change brings back the old settings page and makes it invisible, only accessible using the `?page=jetpack_modules` address.
- When an old version of WordPress is detected, no API endpoints are loaded and a redirect meta tag is added to guide the user to a fallback page.
- Same thing for browsers with no JS - they get redirected to a fallback page.
- In both previous cases the user gets a notice that they should upgrade or turn on JavaScript to see Jetpack in full glory.

#### What doesn't work yet:
- I couldn't figure out why the styles for the fallback page are broken, perhaps @jeffgolenski would know?

### CODE REVIEW

File: _inc/jetpack-modules.views.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ window.jetpackModules.views = (function( window, $, _, Backbone, wp ) {
 					return;
 				}
 
-				var url      = window.location.href.split('?')[0] + '?page=jetpack_modules',
```
Comment (dereksmart) [inline _inc/jetpack-modules.views.js]: Any objections to leaving `jetpack_modules` as the page?  I'm thinking for back-compat if people have some links already pointing to this.

**Comment by dereksmart** [comment]
Is it known that the bulk-select tool isn't working properly?  I mean that clicking this checkbox
screen shot 2016-06-07 at 3 08 08 pm 1
Does not do anything.  

Manually selecting bulk works, however.

**Comment by zinigor** [comment]
Thanks for taking a look @dereksmart! I have fixed the checkbox and reverted the page slug back to `jetpack_modules`.

**Comment by jeffgolenski** [comment]
@zinigor Thanks for building this out!

We just need a few more items taken care of, methinks. 

screen shot 2016-06-08 at 2 42 38 pm

- [ ] Masthead updated to match new masthead (static html version?) currently being redone s bit, do we can hold off on this for now
- [ ] Footer updated to match react footer (static html?)
- [ ] New version of alert notice. should probably be the `is-warning` notice
- [ ] Remove the "Activate Site Management" notice
- [ ] Can we have the "Module info" changed to "feature info"
- [ ] Instead of having "Module info | Activate" only visible on hover, we should keep them always displayed

**Comment by zinigor** [comment]
This PR is now ready for testing. This is how the screen looks like when JS is available but WP version is too old:
screenshot from 2016-07-07 15-27-54

And this is how it looks when there is no JS:
screenshot from 2016-07-07 15-29-01

**Comment by jeherve** [comment]
How is the module order managed? I always wondered why we didn't get an alphabetical order when there was no JS.

**Comment by zinigor** [comment]
Yeah, modules are ordered using Backbone, which is obviously not there when JS is turned off.

**Comment by dereksmart** [comment]
**When JS is enabled and WP version is current**
Styles seem to be really messed up: 
screen shot 2016-07-07 at 12 14 51 pm

**When JS is disabled**
- Page looks ok and I see the notice. 
- Many of the `configure` links do not work, however.  For example: `sharing` or `PBE`
- Activate/deactivate works well
- `feature info` links nowhere

**When WP version is too old**
- Page looks ok and I see the notice. 
- Many of the `configure` links do not work, however.  For example: `sharing` or `PBE`
- Activate/deactivate works well
- Clicking on the title of the module pops open a modal window.  Let's instead remove the old modal code and link to jetpack.com's respective support pages.  This link should already be tied to the module info in `get_module(s)` methods.

**Comment by zinigor** [comment]
I have fixed everything but the broken settings pages, what do you say we handle those in a separate PR since this one is already humongous.

**Comment by dereksmart** [comment]
@zinigor this is looking really good.  The only thing I've noticed is that some mobile views for this page are pretty bad, regardless of js/wp-version/etc. However, I don't see this as a blocker.  It can be fixed when @jeffgolenski / @MichaelArestad  are back from AFK

screen shot 2016-07-08 at 12 09 18 pm

:shipit:

**Pull request merged**